declarations_site/spotter/utils.py

```diff
@@ -255,3 +267,49 @@ def create_notify(task, report):
     send_found_notify(notify)
     notify.save()
     return notify
+
+
+def save_search_task(user, query, deepsearch=True, query_params='', chat_data=''):
+    if len(query) < 2:
+        raise ValueError('Не вдалось створити завдання з пустим запитом.')
+
+    if len(query) > 100 or len(query.split(' ')) > 10:
+        raise ValueError('Не вдалось створити завдання з таким довгим запитом.')
+
+    if not user.email:
+        raise ValueError('Не вдалось створити завдання без адреси електронної пошти.')
+
+    # don't add twice
+    if SearchTask.objects.filter(user=user, query=query,
```

Comment:
unique together is not about indexing, but about adding constrains on your data and keeping it consistent.